### ISSUE TITLE
[persistent storage] Inventory cards shows 0 pvcs/nodes/pvs

### ISSUE THREAD

**Issue by e-minguez** [openshift/console#2488]
![no_data](https://user-images.githubusercontent.com/346758/63700080-5874be80-c822-11e9-9b1a-6215a7320b0e.png)

```
$ oc version
Client Version: version.Info{Major:"4", Minor:"2+", GitVersion:"v4.2.0", GitCommit:"0191acaa1", GitTreeState:"clean", BuildDate:"2019-08-18T17:18:38Z", GoVersion:"go1.12.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.0+f95ec38", GitCommit:"f95ec38", GitTreeState:"clean", BuildDate:"2019-08-22T23:10:14Z", GoVersion:"go1.12.6", Compiler:"gc", Platform:"linux/amd64"}
OpenShift Version: 4.2.0-0.nightly-2019-08-23-034826
$ oc get nodes
NAME                                         STATUS   ROLES           AGE    VERSION
kni1-master-0.example.com   Ready    master,worker   3d1h   v1.14.0+2dbcb37dc
kni1-master-1.example.com   Ready    master,worker   3d1h   v1.14.0+2dbcb37dc
kni1-master-2.example.com   Ready    master,worker   3d1h   v1.14.0+2dbcb37dc
$ oc get pv
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                      STORAGECLASS   REASON   AGE
pvc-02b887a5-c7fd-11e9-9496-ecf4bbed6c4c   5Gi        RWO            Delete           Bound    amq-workload/data-my-cluster-kafka-0       csi-rbd                 147m
pvc-3d2b8960-c7fd-11e9-9496-ecf4bbed6c4c   5Gi        RWO            Delete           Bound    amq-workload/prometheus-claim              csi-rbd                 146m
pvc-4982bc82-c7fd-11e9-ad3d-ecf4bbed6fec   1Gi        RWO            Delete           Bound    amq-workload/grafana-claim                 csi-rbd                 145m
pvc-d193c82e-c7ff-11e9-ad3d-ecf4bbed6fec   60Gi       RWX            Delete           Bound    demo/windows                               csi-rbd                 127m
pvc-d9bd517e-c7fc-11e9-9496-ecf4bbed6c4c   1Gi        RWO            Delete           Bound    amq-workload/data-my-cluster-zookeeper-0   csi-rbd                 148m
pvc-fb3190ac-c7ff-11e9-ad3d-ecf4bbed6fec   11Gi       RWX            Delete           Bound    demo/rhel8-pvc                             csi-rbd                 126m
$ oc get pvc -A
NAMESPACE      NAME                          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
amq-workload   data-my-cluster-kafka-0       Bound    pvc-02b887a5-c7fd-11e9-9496-ecf4bbed6c4c   5Gi        RWO            csi-rbd        147m
amq-workload   data-my-cluster-zookeeper-0   Bound    pvc-d9bd517e-c7fc-11e9-9496-ecf4bbed6c4c   1Gi        RWO            csi-rbd        148m
amq-workload   grafana-claim                 Bound    pvc-4982bc82-c7fd-11e9-ad3d-ecf4bbed6fec   1Gi        RWO            csi-rbd        145m
amq-workload   prometheus-claim              Bound    pvc-3d2b8960-c7fd-11e9-9496-ecf4bbed6c4c   5Gi        RWO            csi-rbd        146m
demo           rhel8-pvc                     Bound    pvc-fb3190ac-c7ff-11e9-ad3d-ecf4bbed6fec   11Gi       RWX            csi-rbd        126m
demo           windows                       Bound    pvc-d193c82e-c7ff-11e9-ad3d-ecf4bbed6fec   60Gi       RWX            csi-rbd        127m
```

**Comment by rawagner** [comment]
AFAIK persistent tab show only PV/PVCs which were provisioned by ceph. @cloudbehl

**Comment by e-minguez** [comment]
> AFAIK persistent tab show only PV/PVCs which were provisioned by ceph. @cloudbehl

This is exactly the case. I only have ceph deployed via the OCS operator.

**Comment by cloudbehl** [comment]
@e-minguez The OCS team has told that Provisioner will not available in PVC YAML always. So we need to look for Storage class name to find PVC Provisioned by rook-ceph.

**Comment by spadgett** [comment]
Fixed by #2329

/close
